style.css

```diff
@@ -1,7 +1,54 @@
+body {
+  display: flex;
+  flex-direction: column;
+  border: #1a2236 solid;
+  background-image: url(images/image_geometry_1.png);
+  background-repeat: no-repeat;
+  margin: 0;
+}
+
+#toolbar {
+  display: flex;
+  justify-content: flex-end;
+  margin: 16px;
+}
+
 h1 {
-    font-family:'Playfair Display' ;
-    font-size: 50px;
-    font-weight: 400;
-    line-height: 100%;
-    width: 200px;
-}
\ No newline at end of file
+  font-family: 'Playfair Display', serif;
+  font-size: 50px;
+  line-height: 100%;
+  width: 200px;
+  padding: 16px;
+  padding-bottom: 0;
+  margin: 0;
+  margin-top: 40px;
```

Comment:
Good job! but I think you mistakenly removed the `font-weight: 400;` property 😊 which is in the original Figma design. Please make sure to return it so that your page becomes much more beautiful.

![image](https://user-images.githubusercontent.com/34779928/199790130-a3ac6c06-a74e-4019-b35b-68bf014f1bae.png)

Reply:
Ups, I will resolve this mistake inmediatly